### ISSUE TITLE
[New Cohort] Linear Algebra - Foundations to Frontiers

### ISSUE THREAD

**Issue by Nixerrr** [ossu/cohorts#6]
## Class

[Computer Science / Core Math / Linear Algebra - Foundations to Frontiers](https://www.edx.org/course/laff-linear-algebra-foundations-to-frontiers)

## Coordinators (2/3)

- [Nikos Panaiotu](https://github.com/Nixerrr)
- [Rhitik Bhatt](https://github.com/lambainsaan)

## Students (6/20)

- [Deepak Kushwaha](https://github.com/netship02)
- [Hassen Ben Tanfous](https://github.com/HassenPy) 
- [David Chen](https://github.com/mvj3)
- [Belhassen Chelbi](https://github.com/belhassen07)

## Start Date

January 23, 2018

## Slack

Join the [OSSU Slack workspace](https://join.slack.com/t/ossuniversity/shared_invite/enQtMjk4NDA5NDAzMzkyLTkwMDQ3MGVjNzYxMDg2ZmIwMmYwZjQ0ZDI1OTdmODdlNDNkZTdlOTZjMWJlMmNhMTZiZjlkNWIzNDliMjEzZTI), then find the channel #laff

**Comment by lambainsaan** [comment]
I'm in can be a coordinator.

**Comment by netship02** [comment]
Deepak kushwaha
As student

**Comment by HassenPy** [comment]
Hello, count me in, Hassen Ben Tanfous as a student.

**Comment by Nixerrr** [comment]
If anyone wants to revise vectors and matrices before the course launches, Khan Academy has material on both: 
- [Vectors ](https://www.khanacademy.org/math/precalculus/vectors-precalc)
- [Matrices](https://www.khanacademy.org/math/precalculus/precalc-matrices)

**Comment by lambainsaan** [comment]
[Essence of Linear Algebra ](https://www.youtube.com/watch?v=kjBOesZCoqc&list=PLZHQObOWTQDPD3MizzM2xVFitgF8hE_ab) has a very intuitive explanation of many complicated Linear Algebra concepts, give it a go. 🤓

**Comment by HassenPy** [comment]
hey guys, i've read this:

> For instance, a coordinator is responsible for setting the pace of a cohort, organizing regular check-ins with the students, and providing any other help the students may need, within reason

now i'm wondering, if i join the cohort, am i bound by a pace? i usually (not necessarily) finish courses in less than 1/2 the time they require, just asking to know how to schedule this, i'm ok with being bound by a pace.

**Comment by Nixerrr** [comment]
@HassenPy You have no obligations at all, and you are welcome to progress at your own pace. I cannot guarantee that I will follow the pace prescribed by the course either. So no problem.

**Comment by mvj3** [comment]
Hello, please add me David Chen as a student. Thx

**Comment by Nixerrr** [comment]
One week left until launch, so I created a [Slack channel](https://ossuniversity.slack.com/messages/C8TMCG25U) for us.

**Comment by HassenPy** [comment]
@Nixerrr, it says "This invite link is no longer active."

**Comment by Nixerrr** [comment]
@HassenPy Sorry about that. Maybe @amorriscode can help us out with that?

**Comment by amorriscode** [comment]
Hey everyone, I made a [Slack invite link that _doesn't_ expire](https://join.slack.com/t/ossuniversity/shared_invite/enQtMjk4NDA5NDAzMzkyLTkwMDQ3MGVjNzYxMDg2ZmIwMmYwZjQ0ZDI1OTdmODdlNDNkZTdlOTZjMWJlMmNhMTZiZjlkNWIzNDliMjEzZTI).

**Comment by Nixerrr** [comment]
Awesome! Thank you!

**Comment by Nixerrr** [comment]
Hey people, the course has launched. Come and join us on Slack so we help and keep each other accountable!

**Comment by amorriscode** [comment]
@Nixerrr How's it going? Should we close this up?

**Comment by mvj3** [comment]
I got the [certificate](https://courses.edx.org/certificates/144759ea1c994ae0afc9c5e8f418319d) when I finished 2/3 percents of this course. And I finally finished it completely a few weeks ago, although I didn't do well on the advanced topics. I think I may still need to take another course [Mathematics for Machine Learning: Linear Algebra](https://www.coursera.org/learn/linear-algebra-machine-learning/) in the next few months.
